python-clickclick/riscv64.patch

```diff
@@ -0,0 +1,13 @@
+diff --git PKGBUILD PKGBUILD
+index dd4af1716..910afe88c 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -8,7 +8,7 @@ url='https://codeberg.org/hjacobs/python-clickclick'
+ license=('Apache')
+ arch=('any')
+ depends=('python-click' 'python-yaml')
+-makedepends=('python-setuptools' 'flake8')
++makedepends=('python-setuptools' 'flake8' 'python-six')
```

Comment:
It seems the project mentions `six` only in setup.py. Probably it should be removed.

Reply:
PR'ed as https://codeberg.org/hjacobs/python-clickclick/pulls/1